pkg/sdn/api/types.go

```diff
@@ -44,7 +44,7 @@ type NetNamespace struct {
 	kapi.ObjectMeta
 
 	NetName string
-	NetID   uint
+	NetID   *uint
```

Comment:
Unless you also change it in v1/types.go to a pointer, not specifying it will come in as 0